File: week-07/day-01/draw-square.js
'use strict';

var lineCount = 6;

// Write a program that draws a
// square like this:
//
//
// %%%%%
// %   %
// %   %
// %   %
// %   %
// %%%%%
//
// The square should have as many lines as lineCount is
var puffer = "";
for (let k = 0; k < lineCount; k++) {
  for (let i = 0; i < lineCount; i++) {
    if (k == 0 || k == lineCount - 1) {
      puffer += "%";
    }
    else {
      if (i == 0 || i == lineCount - 1) {
        puffer += "%";
      }
      else {
        puffer += " ";
      }
    }
  }
  console.log(puffer);
  puffer = '';
}

File: week-02/day-01/draw_square.py
# Write a program that reads a number from the standard input, then draws a
# square like this:
#
#
# %%%%%
# %   %
# %   %
# %   %
# %   %
# %%%%%
#
# The square should have as many lines as the number was
lines = int(input("Number of lines: "))
puffer = ""

for k in range(0, lines):
    for i in range(0, lines):
        if k == 0 or k == lines - 1:
            puffer += "%"
        else:
            if i == 0 or i == lines - 1:
                puffer += "%"
            else:
                puffer += " "
           
    print (puffer + "\n")
    puffer = ""